resources/nexus.sh

```diff
@@ -14,13 +14,12 @@ if [[ ${DEBUG_LOGGING} == true ]]
   cp /resources/conf/logback/logback-access.xml ${NEXUS_HOME}/etc/logback/
 fi
 
-# chown the nexus home directory
-chown -R nexus:nexus ${NEXUS_HOME}
+# Chown the nexus data directory
+chown -R nexus:nexus "${NEXUS_DATA}"
```

Comment:
Might be worth putting in the fix we put in to Nexus 2 to prevent the long recursive chown in case of a lot of data in the volume